tools/tsp-client/README.md

```diff
@@ -0,0 +1,46 @@
+# tsp-client
+
+A simple command line tool for generating TypeSpec clients.
+
+### Usage
+```
+tsp-client [options] <outputDir>
+```
+
+## Options
+
+### <outputDir> (required)
```

Comment:
Can `outputDir` is made as optional and compute the value by parsing tspconfig.yaml as `service-dir` and `package-dir` are specified there?

Reply:
ah so you'd pass the `tspconfig.yaml` in the specs repo directly and then we'd calculate the output location? I suppose this would work okay though we'd need to know the root of where you are generating (e.g. your clone of the language repo) if it wasn't the cwd